src/GraphQL.Conventions/Adapters/Engine/ErrorTransformations/IErrorTransformation.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace GraphQL.Conventions.Adapters.Engine.ErrorTransformations
+{
+    public interface IErrorTransformation
+    {
+        Task<ExecutionErrors> Transform(ExecutionErrors errors);
```

Comment:
Any particular reason why you want to return this as a `Task<>`? Any cases where the error transformation would be asynchronous?

Reply:
I was doubting between the two options, you're right will change it to non-async.